lib/modules/datasource/nuget/v3.ts

```diff
@@ -68,6 +68,12 @@ export async function getResourceUrl(
           ? semver.compare(x.version, y.version)
           : /* istanbul ignore next: hard to test */ 0
       );
+
+    // replace a TypeError in the deconstruction with a custom one
+    if (services.length === 0) {
+      throw new Error(`no ${resourceType} services found`);
+    }
```

Comment:
I chose to throw an exception instead of logging and returning because I didn't want to recreate the logging line in the exception handler below. But maybe that's what ought to happen.

Reply:
I think we should check results before returning and don't throw.

it should finally only a debug message without error stack